### DISCUSSION TITLE
What's the best way to change the font?

### DISCUSSION THREAD

**Discussion by vinceimbat** [jpanther/congo#227] [Q&A]
Is this already doable in the current version of the theme? If so, what's the best way to do it?

**Comment by jpanther** [comment]
Although the theme doesn't bundle multiple font options, you have a couple of ways you could customise this...

Creating a [custom CSS file](https://jpanther.github.io/congo/docs/advanced-customisation/#overriding-the-stylesheet) and overriding the default font family:

```css
/* Change the default font */
html {
    font-family: 'your-font';
```

Don't forget to include fallbacks in case your font is not available on all platforms. You could also use the [extended head partial](https://jpanther.github.io/congo/docs/partials/#head-and-footer) to include your own font assets that can then be referenced in your stylesheet. This would be a way to include Google Fonts or other third-party libraries.

You also have the option of completely building the theme from source and adjusting the Tailwind configuration to use whatever fonts you wish. This is more complicated but the theme documentation has [detailed instructions](https://jpanther.github.io/congo/docs/advanced-customisation/#building-the-theme-css-from-source) on this type of customisation.
